Add selectel registry implementation (#4681)
werf/werf: docs/pages_en/advanced/supported_container_registries.md

werf uses the _GitLab Container Registry API_ or _Docker Registry API_ (depending on the GitLab version) to delete tags.

> Privileges of the temporary CI job token (`$CI_JOB_TOKEN`) are not enough to delete tags. That is why the user have to create a dedicated token in the Access Token section (select the `api` in the Scope section) and [perform authorization](#authorization) using it

### Selectel CRaaS

werf uses the [_Selectel CR API_](https://developers.selectel.ru/docs/selectel-cloud-platform/craas_api/) to delete tags, so you need to set either the _username/password_, _account_ and _vpc_ or _vpcID_ to clean up the container registry.

You can use the following options (or their respective environment variables) to set the said parameters:
- `--repo-selectel-username`
- `--repo-selectel-password`
- `--repo-selectel-account`
- `--repo-selectel-vpc` or
- `--repo-selectel-vpc-id`

**Known limitations**

> 1) Sometimes Selectel drop connection with VPC ID. Try to use VPC name.
> 2) Werf can lie layers to the root of Container registry. But CR api have limitations, so you can not remove this layers with api.
> 3) Low API rate-limit. It can cause cleanup problems with rapid development.

Reviewer: ```suggestion
1) Sometimes, Selectel drop connection with VPC ID. Try to use the VPC name instead.
2) CR API has limitations, so you can not remove layers from the root of the container registry.
3) Low API rate limit. It can cause cleanup problems with rapid development
```
